syz-fuzzer/fuzzer.go

```diff
@@ -142,6 +146,12 @@ func createIPCConfig(features *host.Features, config *ipc.Config) {
 	}
 }
 
+// Gate size controls how deep in the log the last executed by every proc
+// program may be. The intent is to make sure that, given the output log,
+// we always understand what was happening.
+// Judging by the logs collected on syzbot, 30 should be a reasonable figure.
+const gateSize = 30
```

Comment:
I think max with prog.MaxPids. Max pids is larger than 30.
Or max with 2*prog.MaxPids?

Reply:
Why `prog.MaxPids`? I think that maybe `max(defaultGateSize, procs) could make some sense, but still -- the gate size is determined by the log size we capture, it shouldn't matter how many procs we have.

Reply:
If procs is larger than the gate size, then some procs will never work effectively.
A user may think it will speed up things, but increasing beyond gate size won't do anything simply because of the gating.

Reply:
A SYZFATAL in case `procs  > gateSize` might do the job here, we probably don't want to silently increase gateSize then.

Reply:
Reduce prog.MaxPids instead.

Reply:
I wish there were some compile-time assertions..